fpdf/adapters/table_pandas.py

```diff
@@ -0,0 +1,38 @@
+from pandas import MultiIndex
+from fpdf import FPDF
+
+
+class FPDF_pandas(FPDF):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def dataframe(self, df, **kwargs):
```

Comment:
Some docstring would be nice before merging this PR 🙂 (as well as an addition in `CHANGELOG.md`)